src/solo12.cpp

```diff
@@ -222,6 +244,12 @@ bool Solo12::calibrate(const Vector12d& home_offset_rad)
     double search_distance_limit_rad =
         10.0 * (2.0 * M_PI / joint_gear_ratios_(0));
     Vector12d profile_step_size_rad = Vector12d::Constant(0.001);
+
+    // Calibrate the right side of the robot in the "other direction".
+    // Do this by running the calibration of the HAA joints in the other direction.
+    profile_step_size_rad(3) *= -1;
```

Comment:
Nice trick. This way you change the spinning direction. Have you tested it on the robot?